File: Ex1/ex1b.py
def main():
    a, b = 10, 5

    total = 0
    total += a
    print(f"sum after += a: {total}")

    product = 1
    product *= b
    print(f"product after *= b: {product}")

    maximum = a if a > b else b
    print(f"Maximum of {a} and {b} is: {maximum}")

    x, y = 15, 12
    result = x - y if x > y else (y - x if y > x else 0)
    print(f"Absolute difference between {x} and {y} is: {result}")

if __name__ == "__main__":
    main()

File: Ex1/ex1b.c
#include <stdio.h>

int main()
 {
    int a = 10, b = 5;

    // Assignment Operators
    int sum = 0;
    sum += a; 
    printf("sum after += a: %d\n", sum);

    int product = 1;
    product *= b; 
    printf("product after *= b: %d\n", product);

    // Conditional Operator
    int max = (a > b) ? a : b;
    printf("Maximum of %d and %d is: %d\n", a, b, max);

    // Nested Conditional Operator
    int x = 15, y = 12;
    int result = (x > y) ? (x - y) : ((y > x) ? (y - x) : 0);
    printf("Absolute difference between %d and %d is: %d\n", x, y, result);

    return 0;
}
